spec/activerecord-bitemporal/bitemporal_spec.rb

```diff
@@ -1065,26 +1065,53 @@ class EmployeeWithUniquness < Employee
       )
     end
 
-    it "create state-destroy record before _run_destroy_callbacks" do
+    it "create state-destroy record around _run_destroy_callbacks" do
```

Comment:
This description is already does not represent the actual scenario, so I have revised it.
Perhaps the description no longer matches what has already been tested in https://github.com/kufu/activerecord-bitemporal/pull/19.